### PR TITLE
Fix/comments sticky toolbar

### PR DESCRIPTION
**Pull request by MattBudz**
### Summary

Fix the comments input textarea not getting the sticky toolbar

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: app/assets/javascripts/shared/editor_toolbar.js [hunk 1]
```diff
@@ -113,25 +114,25 @@ class EditorToolbar {
       var $inputElement = $(this),
           $toolbarElement = $inputElement.parent().prev(),
           $parentElement = $inputElement.parents('[data-behavior~=editor-field]'),
-          $scrollLimitElement = $parentElement.parents('.textile-wrap').parent(),
-          topOffset;
-      
+          $scrollLimitElement = $('[data-behavior~=textile-wrap]').length ? $('[data-behavior~=textile-wrap]') : $('[data-behavior~=view-content]'),
+          topOffset = $('[data-behavior~=textile-wrap]').length ? $parentElement.find('.editor-toolbar').outerHeight() : 85;
```
Comment (aapomm) [inline app/assets/javascripts/shared/editor_toolbar.js:118]: What is `85` here? This smells like a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) and may be moved to a constant.

File: app/assets/javascripts/shared/editor_toolbar.js [hunk 2]
```diff
@@ -113,25 +114,25 @@ class EditorToolbar {
...
+
       $toolbarElement.css({'opacity': 1, 'visibility': 'visible'});
 
       // set offset to 0 if user is in fullscreen mode.
-      if ($inputElement.parents('.textile-fullscreen').length ? topOffset = 0 : topOffset = $parentElement.find('.editor-toolbar').outerHeight());
+      if ($inputElement.parents('[data-behavior~=textile-fullscreen]').length) { topOffset = 40 };
```
Comment (aapomm) [inline app/assets/javascripts/shared/editor_toolbar.js:123]: Ditto on `40` being a magic number

File: app/assets/javascripts/shared/editor_toolbar.js (hunk 1 above)
Comment (MattBudz) [reply app/assets/javascripts/shared/editor_toolbar.js]: fixed

File: app/assets/javascripts/shared/editor_toolbar.js (hunk 2 above)
Comment (MattBudz) [reply app/assets/javascripts/shared/editor_toolbar.js]: fixed

**Pull request merged**